pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -34,6 +34,12 @@ spec:
             - "--leader-election-lease-duration={{ .LeaderElectionLeaseDuration }}"
             - "--leader-election-renew-deadline={{ .LeaderElectionRenewDeadline }}"
             - "--leader-election-retry-period={{ .LeaderElectionRetryPeriod }}"
+            {{ if .KubeApiBurst }}
+            - "--kube-api-burst={{ .KubeApiBurst }}"
```

Comment:
@YiteGu Do you have a link to docs that explains this feature more? I just found the [kubelet config doc](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/), which says this should be set from the kubelet config file, and that it's deprecated. Is there a doc that shows these params are available to all the sidecar containers where you have added it?
_____
(DEPRECATED: This parameter should be set via the config file specified by the kubelet's --config flag. See [kubelet-config-file](https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/) for more information.)

Reply:
> @YiteGu Do you have a link to docs that explains this feature more? I just found the [kubelet config doc](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/), which says this should be set from the kubelet config file, and that it's deprecated. Is there a doc that shows these params are available to all the sidecar containers where you have added it?
> 
> (DEPRECATED: This parameter should be set via the config file specified by the kubelet's --config flag. See [kubelet-config-file](https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/) for more information.)

1. This is an actual case from a self production environment, you can see below:
<img width="1875" alt="output" src="https://github.com/user-attachments/assets/cd13330a-db22-4ad3-81ce-50263ba2f0f9">

2. There have been no throttling after adjust `kube-api-qps`

3. I have checked these sidecars and they all support setting this parameter, as below:
csi-provisioner: https://github.com/kubernetes-csi/external-provisioner/blob/e45cf27000842dc3e2e949480ed38d6c43f149dc/cmd/csi-provisioner/csi-provisioner.go#L103-L104
csi-resizer: https://github.com/kubernetes-csi/external-resizer/blob/d6100254e3d2c35ba18d4e5474b4f223ac58e0e6/cmd/csi-resizer/main.go#L81-L82
csi-attacher: https://github.com/kubernetes-csi/external-attacher/blob/f30d22b35d45d52d9ca1c2515a0b8119c2d3a4ce/cmd/csi-attacher/main.go#L82-L83
csi-snapshotter: https://github.com/kubernetes-csi/external-snapshotter/blob/c72f087f751abb285a90c4f5bb02df9014d2bc19/cmd/csi-snapshotter/main.go#L79-L80

Reply:
Great, thanks for the the confirmation and all the details.